src/SFA.DAS.ApprenticeCommitments.Web/Services/OuterApi/IOuterApiClient.cs

```diff
@@ -50,5 +51,19 @@ public interface IOuterApiClient
         Task ConfirmApprenticeship(
             [Path] Guid apprenticeid, [Path] long apprenticeshipid, [Path] long commitmentStatementId,
             [Body] ApprenticeshipConfirmationRequest confirmation);
+
+        [Patch("/apprentices/{apprenticeId}/apprenticeships/{apprenticeshipId}/statements/{commitmentStatementId}")]
+        Task UpdateApprenticeship(
+            [Path] Guid apprenticeId, [Path] long apprenticeshipId, [Path] long commitmentStatementId,
+            [Body] JsonPatchDocument<Apprenticeship> patch);
+    }
```

Comment:
Would the name be better as PatchApprenticeship

Reply:
You're right, `PatchApprenticeship` names the method precisely for what it is doing.

The other API calls are named along behaviour lines, rather than the HTTP-verb related technology - `VerifyRegistration` not `PostRegistration`; `ConfirmEmployer`, not `PostEmployerConfirmation`, etc.  That way, at the call site you see the intent of the call:

    public class ConfirmYourIdentityModel
    {
        public void OnPost()
        {
            _registrations.VerifyRegistration(...)
        }
    }

I think if we change this one we should probably change all of them, don't you?